apps/wallet/src/ui/app/pages/home/tokens/TokensDetails.tsx

```diff
@@ -408,23 +407,21 @@ function TokenDetails({ coinType }: TokenDetailsProps) {
 										isDefiWalletEnabled ? 'bg-gradients-graph-cards' : 'bg-hero/5',
 									)}
 								>
-									{accountHasSui ? (
-										<div className="flex flex-col gap-1 items-center">
-											<CoinBalance amount={tokenBalance} type={activeCoinType} />
-										</div>
-									) : (
+									<div className="flex flex-col gap-1 items-center">
+										<CoinBalance amount={tokenBalance} type={activeCoinType} />
+									</div>
+
+									{!accountHasSui && (
 										<div className="flex flex-col gap-5">
 											<div className="flex flex-col flex-nowrap justify-center items-center text-center px-2.5">
-												<SvgSuiTokensStack className="h-14 w-14 text-steel" />
-												<div className="flex flex-col gap-2 justify-center">
-													<Text variant="pBodySmall" color="gray-80" weight="normal">
-														To send transactions on the Sui network, you need SUI in your wallet.
-													</Text>
-												</div>
+												<Text variant="pBodySmall" color="gray-80" weight="normal">
```

Comment:
Is it intentional to show this on all networks? Only mainnet should be better I think and keep the previous wording for everything except mainnet.

Reply:
This was not intentional, should be fixed up now!